example/heroic.example.yml

```diff
@@ -105,6 +106,13 @@ metadata:
     #     clusterName: elasticsearch
     #     # ElasticSearch index.
     #     # @default "heroic"
+    #
+    #     # Dynamically sniff new nodes. Defaults false
+    #     sniff: false
+    #
+    #     How often to sniff for new nodes. Defaults 30 seconds
+    #     nodeSamplerInterval: 30
```

Comment:
The new params should be added to the docs too

Reply:
Added them to the docs. So nice they are on the same branch!